docs/developers/quickstart.mdx

```diff
@@ -156,13 +152,142 @@ contract HelloSource {
 }
 ```
 
+### Compile Contract
+
+Compile the contract with the following command: 
+```bash
+npx hardhat compile
+```
+
+>Note: Hardhat may require you to manually install dependencies for @nomicfoundation/hardhat-toolbox. If you get an error about missing dependencies for that plugin, run the follwing command: 
+```bash
+npm install --save-dev "@nomicfoundation/hardhat-network-helpers@^1.0.0" "@nomicfoundation/hardhat-chai-matchers@^1.0.0" "@nomiclabs/hardhat-ethers@^2.0.0" "@nomiclabs/hardhat-etherscan@^3.0.0" "@types/chai@^4.2.0" "@types/mocha@^9.1.0" "@typechain/ethers-v5@^10.1.0" "@typechain/hardhat@^6.1.2" "solidity-coverage@^0.8.1" "ts-node@>=8.0.0" "typescript@>=4.5.0"
+```
+
+To deploy your contract, update your `hardhat.config` file:
+```js
+require("@nomicfoundation/hardhat-toolbox");
+require('@openzeppelin/hardhat-upgrades');
+// Any file that has require('dotenv').config() statement 
+// will automatically load any variables in the root's .env file.
+require('dotenv').config();
+ 
+module.exports = {
+  solidity: "0.8.17",
+  networks:{
+    goerli:{
+      url: "https://rpc.ankr.com/eth_goerli",
+     // PRIVATE_KEY loaded from .env file
+      accounts: [`0x${process.env.PRIVATE_KEY}`]
+    }
+  }
+};
+```
+
+Update your .env file to only have the following line:
+
+```console
+PRIVATE_KEY=YOUR-PRIVATE-KEY-HERE
```

Comment:
oh this is where the template is - i'd move it up to right after creating `.env`